src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -58,20 +60,26 @@ const DownloadReportPage = ({ runningUnitTest }) => {
   } = useSelector(state => state);
 
   const ccdExtendedFileTypeFlag = useSelector(
-    state =>
-      state.featureToggles[
-        FEATURE_FLAG_NAMES.mhvMedicalRecordsCcdExtendedFileTypes
-      ],
+    state => state.featureToggles?.mhv_medical_records_ccd_extended_file_types,
   );
 
   const [selfEnteredPdfLoading, setSelfEnteredPdfLoading] = useState(false);
   const [successfulSeiDownload, setSuccessfulSeiDownload] = useState(false);
   const [failedSeiDomains, setFailedSeiDomains] = useState([]);
   const [seiPdfGenerationError, setSeiPdfGenerationError] = useState(null);
   const [expandSelfEntered, setExpandSelfEntered] = useState(false);
+  const [generatingCCDV2, setGeneratingCCDV2] = useState(false);
 
   const activeAlert = useAlerts(dispatch);
   const selfEnteredAccordionRef = useRef(null);
+  useAcceleratedData();
+
+  // Determine user's data sources based on facility information
+  const facilities = userProfile?.facilities || [];
+  const hasOHFacilities = facilities.some(f => f.isCerner === true);
+  const hasVistAFacilities = facilities.some(f => f.isCerner === false);
+  const hasOHOnly = hasOHFacilities && !hasVistAFacilities;
+  const hasBothDataSources = hasOHFacilities && hasVistAFacilities;
```

Comment:
`useAcceleratedData();` can be removed here.
But question is whether we can use `useAcceleratedData();` and/or the selectors from platform shared code `~/platform/user/cerner-dsot/selectors`?